README.md

```diff
@@ -45,6 +45,8 @@ For more detailed build instructions, see the Servo book under [Setting up your
 
 ### Android
 
+TODO: NDK 26c
+
 - Ensure that the following environment variables are set:
   - `ANDROID_SDK_ROOT`
   - `ANDROID_NDK_ROOT`: `$ANDROID_SDK_ROOT/ndk/25.2.9519653/`
```

Comment:
@mukilan could you also fix readme here

Reply:
Done in [00b4235](https://github.com/servo/servo/pull/32769/commits/00b423578d8a8d5ac3715ea53d64405552061626)

Reply:
Tnx